internal/olm/client.go

```diff
@@ -72,8 +72,13 @@ func (c Client) InstallVersion(ctx context.Context, version string) (*olmresourc
 	objs := toObjects(resources...)
 
 	status := c.GetObjectsStatus(ctx, objs...)
-	if status.HasExistingResources() {
-		return nil, errors.New("detected existing OLM resources: OLM must be completely uninstalled before installation")
+	installed, err := status.HasInstalledResources()
+	if installed {
+		return nil, errors.New("detected existing OLM resources: OLM must be completely uninstalled before installation" +
+			fmt.Sprintf("\nResources:\n%s", status.String()))
+	} else if err != nil {
+		return nil, errors.New("detected errored OLM resources:" +
```

Comment:
@joelanford wdyt about `"detected errored OLM resources, see resource statuses for more details"` as an alternative error message? I'm hesitant to wrap `err` here since it contains error information for only one resource.